yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
remove this file and the node_modules directory. `yarn install` is ran inside of `client`, not at root level.

Reply:
removed the file and the done_modules directory